CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.10)
```

Comment:
Why is everything in all caps? can we fix it to be lower case?

Reply:
As for older cmake code style, commands are written in UPPER CASE.

Reply:
I can run https://github.com/cheshirekow/cmake_format on the files if I can get push access to this PR.